R/use_template.R

```diff
@@ -47,7 +47,8 @@ use_testthat_template <- function(name) {
   # TODO: add the ability to also pass the arguments for the function or find
   #       them within the code base and ensure that the template includes the
   #       necessary structure for each input argument
-  path <- file.path("tests", "testthat", paste0("test-", name, ".R"))
+  file_name <- gsub("_", "-", name)
```

Comment:
I am wondering if we want all the underscores to be changed to dashes? I am fine with either but this means that some current file names would need to be changed, e.g., test-create_default_parameters.R. I think that we should just leave it however the user puts it in that way there is not a hidden mismatch.

Reply:
I left a comment before seeing your message here. I think some files with underscores in their names need to be updated during the test cleanup to maintain consistency. According to the `use_testthat_template()` documentation, we should use function names with underscores. If developers accidentally use a dash in the name argument, it will still generate the correct file name, but they’ll need to manually correct the function names in the tests to ensure they run successfully.